package.json

```diff
@@ -47,10 +46,6 @@
     }
   },
   "//": "This file is partially managed by code-skeleton. Changes may be overwritten.",
-  "tap": {
-    "coverage": true,
-    "ts": true
-  },
```

Comment:
Does this somehow address the current 100% test coverage  threshold expected ? ^^

Reply:
it does not. config has changed in the latest tap so these settings will cause it to throw an error. 100% coverage is the default now. if you need to lower the threshold you can create a `.taprc` file with 

```yaml
allow-incomplete-coverage: true
```

i'll make sure the .gitignore template gets updated to keep `.taprc` files too